styles/perplexity/catppuccin.user.css

```diff
@@ -108,6 +108,28 @@
       border-color: @surface1;
     }
 
+    // Input box
+    ::placeholder {
+      color: @subtext0;
+    }
+    textarea{
+      caret-color: @accent-color;
+    }
+
+    // Text selections
+    ::selection{
+      background-color: @accent-color;
+      color: @mantle;
+    }
```

Comment:
I don't think we should theme this...

Reply:
This is probably the best way if any to do it, but should be check against code blocks since that is the biggest area for errors.

Reply:
Alright.

Reply:
Their own selection styling doesn't keep the syntax highlighting in codeblocks but I think that's fine?

Reply:
Ooh if they style it that's totally fine. I think I'm just opposed to overriding the default system colors if the website itself isn't.